README.md

```diff
@@ -7,6 +7,8 @@ Uses [Gatsby](https://www.gatsbyjs.org/) to build static site.
 
 ## Installation
 
+Starting in September 2021, we use Artifactory (MongoDB's internal package repository) as our primary source of dependencies. With this comes the requirement to login to Artifactory using some credentials. At the time of updating this guide, we are working out our service account on artifactory and getting our API key, but for now you will need to set the following ENV variables in `~/.bash_profile` (which can be 0bined from a team member) `NPM_AUTH` and `NPM_EMAIL`.
```

Comment:
ok the creds are here, but did we update .npmrc to point to our internal rep ? What is the validation plan to check all the packages we use are now available in internal repo?

Reply:
Ah I should have specified this is coming in #907 as that has the `.npmrc` change. In there we specify the new registry.

The validation plan is as follows:
- Swap `.npmrc` in the repo to point to the new registry
- Ensure automated testing passes (done)
- Ensure manual QA w/ Sheeri and Mike passes (done)
- Ensure no errors installing dependencies

This works because Artifactory works as a proxy to the default NPM registry meaning pakages not on the internal registry just get forwarded to the default NPM registry. Furthermore, the package-lock file specifies the URL of the package as well.